packages/wallets/glow/src/adapter.ts

```diff
@@ -0,0 +1,270 @@
+import {
+    BaseMessageSignerWalletAdapter,
+    EventEmitter,
+    scopePollingDetectionStrategy,
+    SendTransactionOptions,
+    WalletAccountError,
+    WalletConnectionError,
+    WalletDisconnectedError,
+    WalletDisconnectionError,
+    WalletError,
+    WalletName,
+    WalletNotConnectedError,
+    WalletNotReadyError,
+    WalletPublicKeyError,
+    WalletReadyState,
+    WalletSignTransactionError,
+} from '@solana/wallet-adapter-base';
+import {
+    Connection,
+    PublicKey,
+    SendOptions,
+    Transaction,
+    TransactionSignature,
+} from '@solana/web3.js';
+
+interface GlowWalletEvents {
+    connect(...args: unknown[]): unknown;
+    disconnect(...args: unknown[]): unknown;
+}
+
+interface GlowWallet extends EventEmitter<GlowWalletEvents> {
+    isGlow?: boolean;
+    publicKey?: { toBytes(): Uint8Array };
+    isConnected: boolean;
+    signTransaction(transaction: Transaction): Promise<Transaction>;
+    signAllTransactions(transactions: Transaction[]): Promise<Transaction[]>;
+    signAndSendTransaction(
+        transaction: Transaction,
+        options?: SendOptions
+    ): Promise<{ signature: TransactionSignature }>;
+    signMessage(message: Uint8Array): Promise<{ signature: Uint8Array }>;
+    connect(): Promise<void>;
+    disconnect(): Promise<void>;
+}
+
+interface GlowWindow extends Window {
+    solana?: GlowWallet;
```

Comment:
We aren't accepting new adapters that use `window.solana`, and we've also requested that wallets that used this previously add a custom name. This prevents multiple installed wallets from fighting over declaring `window.solana`, which ultimately prevents good detection. Would you be willing to change this to `window.glowWallet` or something similar?

Reply:
Updated to `glowSolana` 👍